tests/integration/targets/vyos_interface/tests/cli/intent.yaml

```diff
@@ -7,17 +7,19 @@
     commands:
       - show lldp neighbors detail
 
-- name: Should we run lldp tests?
-  set_fact:
-    run_lldp_tests: "'PortDescr:    eth0' in neighbors_out.stdout[0]"
+
+# setting a variable dynamically using set_fact isn't working in devel.
+# - name: Should we run lldp tests?
+#  set_fact:
+#    run_lldp_tests: "'PortDescr:    eth0' in neighbors_out.stdout[0]"
 
 - name: Enable LLDP service
-  when: run_lldp_tests
+  when: "'PortDescr:    eth0' in neighbors_out.stdout[0]"
   vyos.vyos.vyos_lldp:
     state: present
 
 - name: Create LLDP configuration
-  when: run_lldp_tests
+  when: "'PortDescr:    eth0' in neighbors_out.stdout[0]"
```

Comment:
can't we just remove the when statement and always run them? In zuul this should always be 'eth0' right?

Reply:
This condition is set to false for most of the tasks. These tasks are meant to be skipped. Please refer [log](https://ca0053db361c72af8a53-29895e2627616c145567264765676983.ssl.cf2.rackcdn.com/46/2a7503b272fb362638e38b7bef6becf82f6ec187/check/ansible-test-network-integration-vyos-network_cli-python36-stable29/2526fdf/job-output.html#l30613). May be we can remove the tasks. I would like to hear it from @rohitthakur2590 if it is alright to remove the tasks, in which the condition is checked. I have not seen them run as the condition is always false.

Reply:
If they are always skipped, I feel the test might not be the best. Just looking at the code, I am having trouble understand what we are actually testing.  Agree, lets see what @rohitthakur2590 says, but it feels like if the task is unused, we delete it and remove the idea of condition tasks for testing.

Reply:
@GomathiselviS  Its look we are skipping these tasks anyway, not sure why its there in the first place.IMO we should 
remove these and get rid of any conditional testing here.